src/QueryEditor.tsx

```diff
@@ -232,15 +239,19 @@ export class QueryEditor extends PureComponent<Props, State> {
     return (
       <div className="sl-notebook-query-editor">
         <Field label="Notebook" className="sl-notebook-selector">
-          <Select
-            options={this.state.notebooks.map(formatNotebookOption)}
-            isLoading={this.state.loadingNotebooks}
-            onChange={this.onNotebookChange}
-            menuPlacement="bottom"
-            maxMenuHeight={200}
-            placeholder="Select notebook"
-            value={selectedNotebook ? formatNotebookOption(selectedNotebook) : undefined}
-          />
+          <InlineField label="Notebook">
```

Comment:
You don't need the InlineField component here since we already have the Field component above it. The AsyncSelect can just go directly in the Field.

Otherwise we duplicate the label:
![image](https://github.com/ni/systemlink-notebook-datasource/assets/9257800/adb35b69-7e65-424a-8f17-2ddea11dcc3e)